app/tests/spec/lib/fxa-client.js

```diff
@@ -550,9 +575,19 @@ function (chai, $, sinon, FxaClient, p, testHelpers, Session, FxaClientWrapper,
           return p.reject(AuthErrors.toError('INCORRECT_PASSWORD'));
         });
 
-        return client.checkPassword(email, 'badpassword')
+        sinon.stub(realClient, 'sessionDestroy', sinon.spy());
+
+        return client.checkPassword(email, password)
```

Comment:
It did. The response to realClient.signIn is mocked in above, so I didn't see a point in having the string literal `badpassword`.